includes/wikia/services/ArticleQualityService.php

```diff
@@ -158,6 +159,18 @@ public function getArticleQuality() {
 			$article = new Article( $title );
 			$parser = $article->getParserOutput();
 
+			if ( !$parser ) {
+				//MAIN-3592
+				WikiaLogger::instance()->error(
+					__METHOD__,
+					[
+						'message' => 'Article::getParserOutput returned false',
+						'articleId' => $this->articleId,
+					]
+				);
+				return null;
```

Comment:
Output for this field will be: "article_quality_i":{"set":null} when null is returned, which will remove AQ field from solr document after indexing. Do we want such behaviour?